.github/workflows/check-bump-interface-version.yml

```
@@ -0,0 +1,25 @@
+name: check-bump-interface-version
+
+on:
+  push:
+    paths: 'stellar-contract-env-common/src/env.rs'
+  pull_request:
+    paths: 'stellar-contract-env-common/src/env.rs'
+
+jobs:
+  check_pr:
+    runs-on: 'ubuntu-20.04'
+    steps:
+    - name: "check stellar-contract-env-common/src/meta.rs was modified"
+      uses: brettcannon/check-for-changed-files@v1
```

It's a security hardening best practice to reference third-party actions with a commit sha so that third-parties can't inject code into our builds at their discretion. Ref: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
```suggestion
      uses: brettcannon/check-for-changed-files@ 551263a821fde32615499b71f9283a7e98fcc657 # v1.0.1
```

Can you update the others too?